code/datums/uplink/devices and tools.dm

```diff
@@ -15,7 +15,7 @@
 	name = "Duct Tape"
 	desc = "A roll of duct tape. changes \"HELP\" into sexy \"mmm\"."
```

Comment:
Christ. While you're in the region could you make this less awful?

Reply:
Wow, that's a description I guess. 
I'll slap a duck joke over it.